apps/console/src/features/private-key-jwt/pages/private-key-jwt-config-edit.tsx

```diff
@@ -0,0 +1,256 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {
+    AlertLevels, IdentifiableComponentInterface
+} from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import {
+    DocumentationLink,
+    GridLayout,
+    PageLayout,
+    useDocumentation
+} from "@wso2is/react-components";
+import React, {
+    FunctionComponent,
+    ReactElement,
+    SyntheticEvent,
+    useEffect,
+    useState
+} from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch } from "react-redux";
+import { Dispatch } from "redux";
+import { Checkbox, CheckboxProps, Grid, Icon, Message } from "semantic-ui-react";
+import { AppConstants, history } from "../../core";
+import { ServerConfigurationsConstants } from "../../server-configurations";
+import { updateJWTConfig, useTokenReuseConfigData } from "../api";
+
+/**
+ * Private key JWT client authentication for OIDC configuration page.
+ *
+ * @param props - Props injected to the component.
+ * @returns Private key JWT client authentication for OIDC configuration page component.
+ */
+export const PrivateKeyJWTConfigEditPage: FunctionComponent<IdentifiableComponentInterface> = (
+    props: IdentifiableComponentInterface
+): ReactElement => {
+    const { [ "data-componentid" ]: componentId } = props;
+
+    const dispatch: Dispatch = useDispatch();
+    const { t } = useTranslation();
+    const { getLink } = useDocumentation();
+    const [ enableTokenReuse, setEnableTokenReuse ] = useState<boolean>(true);
+
+
+    const {
+        data: tokenReuseData,
+        isLoading: isLoading
+    } = useTokenReuseConfigData();
+
+    useEffect(() => {
+
+        if (tokenReuseData) {
+            setEnableTokenReuse(tokenReuseData?.enableTokenReuse);
+        }
+
+    }, []);
+
+
+    /**
+     * Handles back button click event
+     */
+    const handleBackButtonClick = () => {
+        history.push(
+            AppConstants.getPaths()
+                .get("GOVERNANCE_CONNECTOR")
+                .replace(
+                    ":id",
+                    ServerConfigurationsConstants.LOGIN_ATTEMPT_SECURITY_CONNECTOR_CATEGORY_ID
+                )
+        );
+    };
+
+    const resolveConnectorUpdateErrorMessage = (error: any): string => {
```

Comment:
Let's define proper types for `error` parameter if possible

Reply:
fixed with https://github.com/wso2/identity-apps/pull/3786/commits/dd8a9366531a26a1a44411371fe1db17b81925ae

Reply:
This isn't yet resolved. Defining type as `any` is not recommended usually.

Reply:
Done